app/src/main/res/values/strings.xml

```diff
@@ -3,45 +3,45 @@
     <string name="settings_backup_apk_title">App backup</string>
-    <string name="settings_backup_apk_summary">Back up the apps themselves. Otherwise, only app data would get backed up.</string>
-    <string name="settings_backup_apk_dialog_title">Really disable app backup?</string>
-    <string name="settings_backup_apk_dialog_message">Disabled app backup will still back up app data. However, it will not get restored automatically.\n\nYou will need to install all your apps manually while having \"Automatic Restore\" switched on.</string>
+    <string name="settings_backup_apk_summary">Also back up the apps themselves. Otherwise, only app data is backed up.</string>
+    <string name="settings_backup_apk_dialog_title">Really turn off app backup?</string>
+    <string name="settings_backup_apk_dialog_message">Even with app backup off, the app data is backed up. However, it will not get auto-restored.\n\nYou will need to install all your apps manually with \"Auto-restore\" on.</string>
     <string name="settings_backup_apk_dialog_cancel">Cancel</string>
-    <string name="settings_backup_apk_dialog_disable">Disable app backup</string>
+    <string name="settings_backup_apk_dialog_disable">Turn off app backup</string>
     <string name="settings_backup_status_title">Backup status</string>
     <string name="settings_backup_status_summary">Last backup: %1$s</string>
     <string name="settings_backup_exclude_apps">Exclude apps</string>
     <string name="settings_backup_now">Backup now</string>
     <string name="settings_category_storage">Storage backup (experimental)</string>
-    <string name="settings_backup_storage_title">Backup my files</string>
+    <string name="settings_backup_storage_title">Back up my files</string>
     <string name="settings_backup_files_title">Included files and folders</string>
     <string name="settings_backup_files_summary">None</string>
     <string name="settings_backup_recovery_code">Recovery code</string>
     <string name="settings_backup_recovery_code_summary">Verify existing code or generate a new one</string>
     <string name="settings_backup_storage_dialog_title">Experimental feature</string>
     <string name="settings_backup_storage_dialog_message">Backing up files is still experimental and might not work. Do not rely on it for important data.</string>
-    <string name="settings_backup_storage_dialog_ok">Enable anyway</string>
+    <string name="settings_backup_storage_dialog_ok">Turn on anyway</string>
     <string name="settings_backup_storage_battery_optimization">Warning: No automatic backups, because battery optimization is active.</string>
     <string name="settings_backup_new_code_dialog_title">New recovery code required</string>
     <string name="settings_backup_new_code_dialog_message">To continue using app backups, you need to generate a new recovery code.\n\nWe are sorry for the inconvenience.</string>
-    <string name="settings_backup_new_code_code_dialog_ok">New code</string>
```

Comment:
And now it is just "code", whereas before it was "recovery code".
Is this a size restriction?

Reply:
It is for a dialog so possibly, can check.